src/screens/WalletScreen.js

```diff
@@ -37,24 +37,22 @@ export default function WalletScreen() {
 
   useEffect(() => {
     if (!initialized) {
-      initializeWallet()
-        .then(() => {
-          setInitialized(true);
-          getKeyboardHeight()
-            .then(keyboardHeight => {
-              if (keyboardHeight) {
-                updateKeyboardHeight(keyboardHeight);
-              }
-            })
-            .catch(() => {
-              setInitialized(true);
-            });
+      initializeWallet();
```

Comment:
Do you wanna keep the try & catch here? If initializeWallet throws we might get in a broken state, even tho part of the app might still be usable